README.md

```diff
@@ -3,14 +3,14 @@
 This project contains several reusable Github workflows meant to be included in Ledger embedded applications repositories.
 
 Several of them are mandatory in order to ensure a level of quality and standardisation among all applications.
-The mandatory workflows must pass for an application to be deployed on the Ledger store. 
+The mandatory workflows must pass for an application to be deployed on the Ledger store.
 
 Others are optional and are provided here to help developers create their own application for Ledger devices.
 
 ## Description of available workflows
 
-- `app_compliance.yml`\
-	This workflow is mandatory, itwill call several child reusable workflows.
+- `reusable_guidelines_enforcer.yml`\
+	This workflow is **mandatory**. It will call several child reusable workflows.
```

Comment:
+ add the device selection option
+ add a small explanation that if the dev disables the workflow for a device we won't deploy it on the store for this device